control-plane/tests/test_service_persistence_ingest_case_lifecycle.py

```
@@ -349,6 +390,94 @@ def test_service_delegates_detection_intake_and_triage_operations(self) -> None:
             ),
             attach_result,
         )
+        self.assertIs(
+            service.ingest_wazuh_alert(
+                raw_alert={"alert": {"data": {"source_family": "github_audit"}}},
+                authorization_header="Bearer reviewed-shared-secret",
+                forwarded_proto="https",
+                reverse_proxy_secret_header="reviewed-proxy-secret",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "reverse_proxy_secret_header"

(S106)